javascript-functions/main.js

```diff
@@ -0,0 +1,29 @@
+function convertMinutesToSeconds(minutes) {
+  var seconds = minutes * 60;
+  console.log('Minutes to seconds:', seconds);
+}
+convertMinutesToSeconds(5);
```

Comment:
So for each function, what we want to do is to return the value (so it's called `return function`) and then store that value into a variable and then console log it outside of each function. Does that make sense? If we don't return any value inside of function, then that function becomes a `void function`